package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "homeday-blocks",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
Should be correct now. :+1:

Reply:
Hey @SinisaG you can use [`npm version`](https://docs.npmjs.com/cli/version) in this case. It automatically bumps `package.json` and `package-lock.json` for example. You can check an example here: https://github.com/VueFloripa/vuefloripa.github.io/blob/development/package.json#L8

With this merge we got a mismatch between both files.